tests/python/tests/http_server/php/index.php

```diff
@@ -119,6 +119,25 @@ public function work() {
         flush();
         sleep(2);
         throw new Exception('Exception');
+     case "flush_and_header_register_callback_flush_inside_callback":
+        echo "Zero ";
+        header_register_callback(function () {
+            echo "Two ";
+            flush();
+            sleep(2);
+            echo "Three ";
+        });
+        echo "One ";
+        return;
+     case 'flush_and_header_register_callback_callback_after_flush':
+        echo "Zero ";
+        header_register_callback(function () {
+        	echo "Unreachable ";
```

Comment:
As far as I can see it must be reachable here

Reply:
@DrDet 
Yes, I agree, fixed